### PR TITLE
Fix typos in routes and system sections of guide

### PR DESCRIPTION
**Pull request by tflem**

### CODE REVIEW

File: getting-started/routing/routes.md [hunk 1]
```diff
@@ -20,16 +20,16 @@ end
...
-Lets say you are defining a static website and you want all your URL to be displayed as `http://www.mycoolsite.com/page` you will define your routes as:
+Lets say you are defining a static website and you want the URL to be displayed as `http://www.mycoolsite.com/page`. You will define your routes as:
 
 ```crystal
 # routes(pipeline, scope)
 routes :web, "/page"
 ```
 
-The routes macro takes a last argument, a block, within the block is where you define your routes.
+The routes macro takes a last argument and a block, within the block is where you define your routes.
```
Comment (robacarp) [inline getting-started/routing/routes.md]: This could probably be a period instead of a comma.

File: getting-started/testing/system.md [hunk 2]
```diff
@@ -42,7 +42,7 @@ Each environment's configuration can be modified similarly. In this case, we can
 
 **Setting Up System Specs**
 
-We have made it as simple as possible to have your system specs. Before running your specs ensure you have installed the `chromedriver` and `selenium standalone server` is in your system path. 
+We have made it as simple as possible to have your system specs. Before running your specs ensure you have installed the `chromedriver` and  that `selenium standalone server` is in your system path.
```
Comment (robacarp) [inline getting-started/testing/system.md]: The whole `system specs` section needs a lot of attention, but `selenium standalone server` is the wrong command. The correct command is `selenium-server standalone`.

File: getting-started/routing/routes.md (hunk 1 above)
Comment (tflem) [reply getting-started/routing/routes.md]: You're welcome! Glad to help. Changes made.

**Comment by robacarp** [comment]
Thanks @tflem

**Pull request merged**